src/client/game/nav.ts

```diff
@@ -55,9 +55,17 @@ export function registerInviteBtnListener(f: () => void) {
   };
 }
 
-export function registerRestartButtonListener(f: () => void) {
+export function registerRestartBtnListener(f: () => void) {
   const restartBtn = document.getElementById("restart");
-  restartBtn.classList.remove("hidden");
+  restartBtn.classList.remove("invisible");
+  restartBtn.onclick = () => {
+    f();
+  };
+}
+
+export function registerMuteAllBtnListener(f: () => void) {
+  const restartBtn = document.getElementById("muteAll");
```

Comment:
should this variable be named `muteBtn`?

Reply:
Definitely, updating